embedchain/vectordb/chroma_db.py

```diff
@@ -35,9 +35,11 @@ def _get_or_create_db(self):
         """Get or create the database."""
         return chromadb.Client(self.client_settings)
 
-    def _get_or_create_collection(self):
+    def _get_or_create_collection(self, name=None):
         """Get or create the collection."""
+        if name is None:
+            name = "embedchain_store"
         return self.client.get_or_create_collection(
-            "embedchain_store",
+            name=name,
```

Comment:
can you point me where the default value of collection is getting set?

Reply:
Line 41, if nothing is passed as a collection name it will set it to "embedchain_store"

Reply:
I recommend you merge PR #310 first, it's dedicated to this issue @taranjeet